### DISCUSSION TITLE
Migrate SEQ to new server

### DISCUSSION THREAD

**Discussion by jdmcfadden-usx** [datalust/seq-tickets#1720] [Help]
We are replacing the SEQ server with a new virtual machine. What is the best way to move customizations such as API keys, users, signals, etc., to the new machine? I would prefer not to have to set everything up all over again if I can help it.

**Comment by nblumhardt** [comment]
Hi! Great to hear your Seq instance is getting some shiny new hardware :-D ... If you want some help picking optimal specs, please let us know via support@datalust.co; any performance related info you can include (possibly a diagnostic report from Settings > Diagnostics) would help us narrow it down for you.

The best way to get configuration across is via backup and restore, which is described step by step in:

https://docs.datalust.co/docs/backup-and-restore

Let me know if this helps,
Nick
